pilot/pkg/xds/discovery.go

```diff
@@ -238,8 +238,6 @@ func (s *DiscoveryServer) periodicRefreshMetrics(stopCh <-chan struct{}) {
 		select {
 		case <-ticker.C:
 			push := s.globalPushContext()
-			push.Mutex.Lock()
-
```

Comment:
Do we need the LastPushMutex?

Reply:
// OnConfigChange is called when a config change is detected.
func (ps *PushContext) OnConfigChange() {
	LastPushMutex.Lock()
	LastPushStatus = ps
	LastPushMutex.Unlock()
	ps.UpdateMetrics()
}